hack/go-gen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+REPO_NAME=$(basename "${PWD}")
+if [ "$IS_CONTAINER" != "" ]; then
+  go generate ./pkg/apis/...
```

Comment:
Running this line forcefully in a container, as the `goimports` tool is a requirement for the environment, but a user/CI should not nessesary have it.

Reply:
I think we may be able to workaround this by running it from the vendor directory, do you know where the go generate command is in the code that is causing us to need `goimports`?